### DISCUSSION TITLE
File name with `(` not allowed.

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo_api#172] [General]
I  was uploading a file named `videoplayback (3).mp4`, but can't upload it . Receives an error `File name with `(` not allowed.`. 
Is there a special reason to filter the filename with `(`, because people have multiple files with the same name with prefix.

And in Linux , it is stored the same way my file was. 

So, if there is no special reason then this can be corrected.

**Comment by dantownsend** [comment]
At the moment it's quite strict with the allowed characters:

https://github.com/piccolo-orm/piccolo_api/blob/ef105f7536373039d2d92faa73fc1618e79c8583/piccolo_api/media/base.py#L63-L70

You can override it using the `allowed_characters` parameter.

However, I think we should just add `(` and `)` to `ALLOWED_CHARACTERS`, because as you say it's very common.

**Reply by sumitsharansatsangi** [reply]
I have added '(' and ')' as allowed character.  see [here](https://github.com/piccolo-orm/piccolo_api/pull/171/commits/f92be81acceaf91508e8d30e05c9e1253d0c6d47)

**Reply by dantownsend** [reply]
Great, thanks!

**Comment by dantownsend** [comment]
Fixed in 0.43.0
